app/src/components/LayerControlBody.vue

```diff
@@ -296,7 +353,12 @@ export default {
       scaleMax: 10,
       scaleStep: 0.01,
 
-      testValue: 0
+      testValue: 0,
+
+      selectedResolutionX: 0,
+      selectedResolutionY: 0,
+      selectedResolutionZ: 0,
+      voxelSpace: true
```

Comment:
variable should be more specific. either `useVoxelSpace` or `useVoxelSpaceFlag` to indicate  it is a boolean. 

currently, this variable can be misunderstood as a custom coordinate space